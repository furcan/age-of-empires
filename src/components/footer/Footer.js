import './Footer.scss';

function Footer() {
  return (
    <footer className="aoe-footer">
      <p>&copy; 2021 Age of Empires. All rights reserved.</p>
    </footer>
  );
}

export default Footer;
