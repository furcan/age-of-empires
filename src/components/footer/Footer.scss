@import "../../styles/variables";

.aoe-footer {
  width: 100%;
  background: $dark;
  color: $light;
  padding: 40px 20px;

  p {
    margin: 0;
    padding: 0;
    font-size: 12px;
    text-align: center;
  }
}
