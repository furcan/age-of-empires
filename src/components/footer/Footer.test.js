
import renderer from 'react-test-renderer';

import Footer from './Footer';

it('The Footer component has been rendered as expected.', () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
