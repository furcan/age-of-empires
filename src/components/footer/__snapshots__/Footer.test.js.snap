// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Footer component has been rendered as expected. 1`] = `
<footer
  className="aoe-footer"
>
  <p>
    Â© 2021 Age of Empires. All rights reserved.
  </p>
</footer>
`;
